app/javascript/components/App.js

```diff
@@ -20,6 +28,7 @@ class App extends Component {
         <Routes>
           <Route exact path="/" element={<Home />} />
           <Route path="/" element={<Header />} />
+          <Route path="/apartindex" element={<ApartIndex apts={this.state.apts} />} />
           <Route exact path="/" element={<Footer />} />
```

Comment:
just a heads up with this routing set up footer should only show up on the landing page. Header and footer may want to be moved where they show up regardless of what route you are on.  Doesn't have to be this PR but in general something you may want to look at and make a change.

Reply:
Definitely something I will change moving forward.  Thanks